ui_framework/doc_site/src/views/button/button_with_icon.js

```diff
@@ -80,7 +80,7 @@ export default () => (
 
     <KuiButton
       iconReverse
-      icon="arrowRight"
+      iconType="arrowRight"
```

Comment:
Let's use this term of putting "type" in a prop name when we're passing in a string reference to a type.